test/system/110-history.bats

```diff
@@ -55,4 +55,16 @@ size      | -\\\?[0-9]\\\+
 
 }
 
+@test "podman image history Created" {
+    run_podman image list --format '{{.CreatedSince}}' $IMAGE
+    imageSince="$output"
+    run_podman image list --format '{{.CreatedAt}}' $IMAGE
+    imageAt=$output
+    run_podman image history $IMAGE
+    run_podman image history --format '{{.CreatedSince}}' $IMAGE
+    is "${lines[0]}" "$imageSince" "History image list should match .CreatedSince"
+    run_podman image history --format '{{.CreatedAt}}' $IMAGE
+    assert "${lines[0]}"  == "$imageAt" "History image list should match .CreatedAt"
```

Comment:
LGTM but should you need to re-push for any reason, a small suggestion that reduces the number of podman calls (hence reduces CI time) and also runs sanity checks on the time values
```bash
@test "podman image history Created" {
    # Values from image LIST
    run_podman image list --format '{{.CreatedSince}}--{{.CreatedAt}}' $IMAGE
    from_imagelist="$output"
    assert "$from_imagelist" =~ "^[0-9].* ago--[0-9]+-[0-9]+-[0-9]+ [0-9:]+ " \
           "CreatedSince and CreatedAt look reasonable"

    # Values from image HISTORY
    run_podman image history --format '{{.CreatedSince}}--{{.CreatedAt}}' $IMAGE
    assert "${lines[0]}" == "$from_imagelist" \
           "CreatedSince and CreatedAt from image history should == image list"
}
```